packages/core/admin/admin/src/content-manager/components/AttributeFilter/index.js

```diff
@@ -1,15 +1,83 @@
 import React from 'react';
 
+import { useQueryParams, getDisplayName } from '@strapi/helper-plugin';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 
+import { useAdminUsers } from '../../../hooks/useAdminUsers';
+
+import { AdminUsersFilter } from './AdminUsersFilter';
 import Filters from './Filters';
 import useAllowedAttributes from './hooks/useAllowedAttributes';
 
+const AUTHOR_ATTRIBUTES = ['createdBy', 'updatedBy'];
+
 const AttributeFilter = ({ contentType, slug, metadatas }) => {
   const { formatMessage } = useIntl();
+
+  const [{ query }] = useQueryParams();
+  // We get the users selected' ids
+  const selectedUsers =
+    query?.filters?.$and?.reduce((acc, filter) => {
+      const [key, value] = Object.entries(filter)[0];
+      const id = value.id?.$eq || value.id?.$ne;
+
+      if (AUTHOR_ATTRIBUTES.includes(key) && !acc.includes(id)) {
+        acc.push(id);
+      }
+
+      return acc;
+    }, []) ?? [];
+  const { users, isLoading } = useAdminUsers(
+    { filter: { id: { in: selectedUsers } } },
+    {
+      enabled: selectedUsers.length > 0,
+    }
+  );
+
   const allowedAttributes = useAllowedAttributes(contentType, slug);
   const displayedFilters = allowedAttributes.map((name) => {
+    if (
+      contentType.attributes[name].type === 'relation' &&
+      contentType.attributes[name].target === 'admin::user'
+    ) {
```

Comment:
Do you think we need both checks? `target === 'admin::user'` seems sufficient for me.

Reply:
does this affect to all relations to admin::users? there might be more relations pointing to that Content Type (Review workflow Assignee, other people custom relations...)